web/docs/advanced/accessing-app-config.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Accessing the configuration 
+---
+
+Whenever you start a Waps app, you are starting two processes.
+ - **The client process** - A React app that implements your app's frontend.
+
+    During development, this is a dev server with hot reloading. In production,
+    it's a Node process that serves pre-built static files with environment variables
+    embedded during the build.
+
+ - **The server process** - An Express server that implements your app's backend.
+ 
+  During development, this is an Express server controlled by a
+  [`nodemon`](https://www.npmjs.com/package/nodemon) process that takes care of
+  hot reloading and restarts. In production, it's a regular Express server run
+  using Node.
+
+Check [the introduction](/docs) for a more in-depth explanation of Wasp's runtime architecture.
+
+You can configure both processes through environment variables. See [the
+deployment instructions](/docs/advanced/deployment/manually#environment-variables) for a full list
```

Comment:
We should make a better list. Here's an issue that tracks it: https://github.com/wasp-lang/wasp/issues/1894